test/ssl-tests/04-client_auth.conf

```diff
@@ -543,7 +543,7 @@ client = 18-client-auth-TLSv1.2-require-client
 [18-client-auth-TLSv1.2-require-server]
 Certificate = ${ENV::TEST_CERTS_DIR}/servercert.pem
 CipherString = DEFAULT
-ClientSignatureAlgorithms = SHA256+RSA
+ClientSignatureAlgorithms = SHA256+PSS
```

Comment:
This looks odd. There doesn't seem to be a corresponding change in 04-client_auth.conf.in for this.